tests/unit/CorrectCompanyName.spec.ts

```diff
@@ -1,36 +1,45 @@
+import Vue from 'vue'
 import Vuetify from 'vuetify'
 import flushPromises from 'flush-promises'
 import { mount, Wrapper } from '@vue/test-utils'
 import CorrectCompanyName from '@/components/correct-name/CorrectCompanyName.vue'
+import { CorrectNameOptions } from '@bcrs-shared-components/enums'
+
+const vuetify = new Vuetify({})
+
+const inputSelector = '#company-name-textfield'
 
 function getLastEvent (wrapper: Wrapper<CorrectCompanyName>, name: string): any {
-  const eventsList = wrapper.emitted(name) as Array<any>
+  const eventsList: Array<any> = wrapper.emitted(name)
   if (eventsList) {
-    const events = eventsList[eventsList.length - 1] as Array<any>
+    const events: Array<any> = eventsList[eventsList.length - 1]
     return events[0]
   }
   return null
 }
 
-describe.skip('CorrectCompanyName', () => {
-  let vuetify: any
+const defaultProps = {
+  companyName: 'Old Company Name',
+  formType: '',
```

Comment:
Form type is an enum -- you should not initialize it as a string. Use null if needed.